migration/migrator/migrations/system/20190510173527_jwt_session_secret.py

```diff
@@ -0,0 +1,26 @@
+"""Migration for the Submitty system."""
+import json
+from pathlib import Path
+import secrets
+import shutil
+import string
+
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    if 'php_user' in config.submitty_users:
```

Comment:
this is to check if the machine is a worker machine or not (if it is, it won't have this user defined)